intro-to-tests/automated-tests.md

```diff
@@ -66,21 +75,46 @@ Because there are a lot of different kinds of code, there are a lot of different
 
 Addison could come up with the following scenarios:
 
 | Question to Ask                                                             | Specific Arguments                          | Expected Return Value       |
 | --------------------------------------------------------------------------- | ------------------------------------------- | --------------------------- |
-| Do we get an accurate birth chart if we pass in a date, location, and time? | `"January 1, 1980"`, `"Seattle"`, `"12:00"` | `{"Sun": "Capricorn", ...}` |
-| What happens if the function receives `None` for time?                      | `"January 1, 1980"`, `"Seattle"`, `None`    | `None`                      |
-| What happens if the `date` argument comes as a different format?            | `"1980-01-01"`, `"Seattle"`, `"12:00"`      | `{"Sun": "Capricorn", ...}` |
-| What happens if the function receives an empty string for `location`?       | `"January 1, 1980"`, `""`, `"12:00"` | should raise an exception   |
+| 1. Do we get an accurate birth chart if we pass in a date, location, and time? | `"January 1, 1980"`, `"Seattle"`, `"12:00"` | `{"Sun": "Capricorn", ...}` |
+| 2. What happens if the function receives `None` for time?                      | `"January 1, 1980"`, `"Seattle"`, `None`    | `None`                      |
+| 3. What happens if the `date` argument comes as a different format?            | `"1980-01-01"`, `"Seattle"`, `"12:00"`      | `{"Sun": "Capricorn", ...}` |
+| 4. What happens if the function receives an invalid place for `location`?       | `"January 1, 1980"`, `"xyz"`, `"12:00"` | should raise an exception   |
+
+To ensure that our test cases cover all scenarios, it can be helpful to categorize tests using the following categories:
+1. Positive Nominal
+1. Negative Nominal
+1. Positive Edge
+1. Negative Edge
+
+#### Nominal Test Cases
+
+A nominal test case is a type of test case that describes a piece of core functionality needed for the success of this method. This is the test case that verifies that the method does its primary responsibility. Nominal test cases are the obvious test cases.
+
+In the case of Addison's astrological software, the first test case listed above is a nominal **positive nominal** test case. It checks that given the correct, expected input (the `date`, `location`, and `time`), the functions returns the correct output (a dictionary with the sun, rising, and moon signs). 
+
+A **negative nominal** test case is a type of test case that describes the most obvious, most typical way that a function handles unexpected input. Again, consider our test cases outlined above, the second test case falls under this category. If we pass in `None` for time, the function
+does not have enough information to determine the astrological birth chart, and as such, it returns `None`.
```

Comment:
I've never heard of a negative nominal case, and I'm not seeing it coming up in Google searches either. To me, `None` feels like an edge case, and we can remove discussion of negative nominal entirely.